index.php

```diff
@@ -60,15 +76,108 @@ function dataDisplay($dataInput)
         <a href="log_book.php"><button>Log Book</button></a>
     </div>
 </section>
+<?php
```

Comment:
There's a simpler way of organising this sorting code, which I'll explain in person